client/src/components/AuthClient.ts

```diff
@@ -118,9 +118,13 @@ const verifyEmail = async () => {
 /**
  * Sends request to resend the account verification link to the user's email
  */
-const resendVerifyEmail = async () => {
+const resendVerifyEmail = async (token?: string) => {
```

Comment:
optional token would only be included from the "verification link has expired" flow